src/app/plugins/platform/Docker/plugins/cms/Drupal8/installDrupal.ts

```diff
@@ -120,4 +114,84 @@ async function installDrupal({
   }
 }
 
+/**
+ * Rename the generated web root if necessary.
+ *
+ * @param {InstallDrupalOptions} {
```

Comment:
Useless JSDoc comment. I think TypeDoc calls this `__namedParameters` or something - it's a bit odd. We should investigate coding standards for this pattern and its implication on documentation.